_data/sites/ilithya.json

```diff
@@ -0,0 +1,6 @@
+{
+	"url": "https://www.ilithya.rocks/",
+	"name": "ilithya",
+	"description": "Personal site of a creative developer, designer and digital artist.",
+	"twitter": "ilithya_net",
```

Comment:
Remove the trailing comma here.

Reply:
OMG, thank you!

I thought it was something more complex 🙃

Reply:
You're welcome.

Did you pushed a new commit? I don't see the checks being rerun :o

Reply:
Nope, not yet. I am slow -.-

I never use github - lol. Only git, or other tools, so rn here in the browser I am a bit clueless.

Reply:
Do you want to apply the patch on GitHub? Or shall I give you some code to check it out with git?

Reply:
Oh, there it is! :crossed_fingers: that it passes now.

No worries about being slow. Take your time :-)
I expected the comment to be made after the change, that's all.

Reply:
I did it now! I am slow, but I do get it usually in the end 🙃

Dude, you were typing super fast! - I am falling asleep here... zzz

Reply:
Yeih 🥳 
Thank you so much for replying!

I was going crazy for a few days, thinking my netlify setup was wrong or smth, but my site is running smooth.

I did try to read some logs when I clicked on _details_ of _header rules_, but it was such a long log, I didn't really look.